src/localizations/en.json

```diff
@@ -107,14 +112,24 @@
     "search_label": "Search after form or service",
     "filter_label": "Filter on service owner",
     "search_hits": "matches found:",
-    "missing_role_access": "The user does not have any required role(s) for the reportee party. (urn:altinn:rolecode:HADM, urn:altinn:location:garage), would give access to delegate the service.",
-    "missing_delegation_access": "The user does not have the necessary delegation accesses to delegate this service.",
+    "missing_role_access": "You don't have any required role(s) for the reportee party. (urn:altinn:rolecode:HADM, urn:altinn:location:garage), would give access to delegate the service.",
+    "missing_delegation_access": "You don't have the necessary delegation accesses to delegate this service.",
+    "missing_srr_right_access": "You don't have SRR right access to delegate",
     "unknown": "Unknown",
     "new_error": "This is a new error that we haven't gotten before. Please report it to Altinn by sending in a screen shot.",
     "missing_role_access_title": "Mangler rolle",
     "missing_delegation_access_title": "Mangler delegeringsrettighet",
     "unknown_title": "Unknown error",
     "new_error_title": "New error",
-    "missing_delegable_right": "Missing delegable rights"
+    "missing_srr_right_access_title": "Missing SRR right access",
+    "missing_delegable_right": "Missing delegable rights",
+    "confirm_delegation_text": "Confirm that you want to give these rights to {{name}}",
+    "choose_rights_chip_text": "Choose rights you want to give access to below:",
+    "choose_service_page_top_text": "Her kan du gi tilgang til skjema og tjenester til {{name}}. Vi anbefaler å filtrere på skjemaeier, og søke med navn på tjenesten. Legg til ved å velge fra listen under.",
```

Comment:
Didn't notice it before, but maybe this should be translated? 😅

Reply:
Fixed